OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidICCAnalysis.scala

```diff
@@ -0,0 +1,56 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+package android
+
+import org.opalj.br.{ClassFile, ObjectType}
+
+import scala.collection.mutable.ListBuffer
+
+
+/**
+ * This class is used to reconstruct intent filters from the analysed project. It holds all relevant information to
+ * simulate intent matching.
+ *
+ * @author Tom Nikisch
+ */
+class IntentFilter(var receiver: ClassFile, var componentType: String) {
+
+
+    final val addDataAuthority = "addDataAuthority"
```

Comment:
No idea why these are all defined explicitly. Also, not all of them seem to be used at all.